bat/BLDDIST.BAT

```diff
@@ -0,0 +1,79 @@
+@if /I "%1" == "tron" (echo on) else (echo off)
+rem Script to create z390 distribution zip file
+
+setlocal
+if /I "%1" == "tron" (set z_TraceMode=tron
+                      shift /1
+              ) else (if /I "%1" == "troff" (set z_TraceMode=troff
+                                             shift /1
+                                     ) else (set z_TraceMode=)
+                      )
+set /A z_NestLevel=%z_NestLevel%+1
+rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
+pushd %~dps0..
+
+rem Get version from version.txt
+set /p VERSION=<version.txt
+echo Building z390 %VERSION%
+set DIST_DIR=dist
+
+rem build jar
+call bat\BLDJAR.BAT %z_TraceMode%        || goto error
+
+rem generate HLASM compatible SPM's
+call bat\BLDZSTRMAC.BAT %z_TraceMode% build\zstrmac || goto error
+
+rem build standard libraries
+call bat\BLDLIB.BAT %z_TraceMode%        || goto error
+call bat\BLDCBLLIB.BAT %z_TraceMode%     || goto error
+
+rem Create/clean dist folder
+if not exist %DIST_DIR% mkdir %DIST_DIR%
+echo /** > %DIST_DIR%\.gitignore
+if exist %DIST_DIR%\z390_%VERSION% rmdir /S /Q %DIST_DIR%\z390_%VERSION%
+if exist %DIST_DIR%\z390_%VERSION%.zip del /F /Q %DIST_DIR%\z390_%VERSION%.zip
+
+cd %DIST_DIR%
+mkdir z390_%VERSION%
+cd z390_%VERSION%
+
+echo on
```

Comment:
Could you please remove this statement? echo can be turned on or off with the tron/troff argument on the invocation of the script.

Reply:
ok